cmd/snap/cmd_recovery_test.go

```diff
@@ -40,8 +40,10 @@ The recovery command lists the available recovery systems.
                                       some things. (default: auto)
       --unicode=[auto|never|always]   Use a little bit of Unicode to improve
                                       legibility. (default: auto)
-      --show-recovery-keys            Show recovery keys (if available) to
-                                      unlock an encrypted partitions.
+      --show-keys                     Displays recovery keys that can be used
+                                      to unlock the encrypted partitions if the
+                                      device-specific automatic unlocking does
+                                      not work.
```

Comment:
this now is very long

Reply:
my suggestion was to have this kind of long text in the:

```
The recovery command lists the available recovery systems.

With --show-keys it displays recovery keys that can be used to unlock the encrypted partitions if the device-specific automatic unlocking does not work.
```

And keep a shorter description for the option itself.